### DISCUSSION TITLE
May i join your team

### DISCUSSION THREAD

**Discussion by Shreedhan003** [ROM-builders/temporary#10233] [General]
https://cirrus-ci.com/task/5694040438996992?logs=build#L4861

**Comment by Shreedhan003** [comment]
Device Specific DT- https://github.com/VoltageOS-Devices/android_device_xiaomi_lavender
Common DT- https://github.com/VoltageOS-Devices/android_device_xiaomi_sdm660-common
Kernel- https://github.com/VoltageOS-Devices/android_kernel_xiaomi_lavender
Vendor- https://github.com/VoltageOS-Devices/android_vendor_xiaomi_lavender
Common Vendor- https://github.com/VoltageOS-Devices/android_vendor_xiaomi_sdm660-common

**Comment by Apon77** [comment]
> https://cirrus-ci.com/task/5694040438996992?logs=build#L4861

give telegram id

**Reply by Shreedhan003** [reply]
Sorry for late reply 
Telegram ID - @xtra_sharif

**Reply by Shreedhan003** [reply]
@Apon77 sir waiting for your reply

**Reply by Apon77** [reply]
> Sorry for late reply Telegram ID - @xtra_sharif

mismatched username/name (github and telegram) cant be added because i cant mention you later in telegram

**Reply by Shreedhan003** [reply]
sir I can't change my github username to exact telegram user name i.e. "xtra_sharif" as "_" not accepted in github username and telegram don't accept "-" in username, so if I change github username to"xtra-sharif" will that be enough to join team ??

**Reply by Apon77** [reply]
> sir I can't change my github username to exact telegram user name i.e. "xtra_sharif" as "_" not accepted in github username and telegram don't accept "-" in username, so if I change github username to"xtra-sharif" will that be enough to join team ??

hmm

**Reply by Shreedhan003** [reply]
So should I change to xtra-sharif ??

**Reply by Apon77** [reply]
> So should I change to xtra-sharif ??

yes

**Reply by Shreedhan003** [reply]
Ok changed username you can proceed
